ads/google/a4a/traffic-experiments.js

```diff
@@ -71,17 +76,22 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
     const isSetFromUrl = maybeSetExperimentFromUrl(win, element,
         experimentName, externalBranches.control,
         externalBranches.experiment, MANUAL_EXPERIMENT_ID);
-    const experimentInfo = {};
-    experimentInfo[experimentName] = internalBranches;
+    const experimentInfoMap = {};
+    experimentInfoMap[experimentName] = {
```

Comment:
I started doing that, but the logic on line 105 that decides whether or not A4A is enabled, needs to know whether the chosen experiment was the control or the experiment, so it's not completely trivial to change it to using anonymous branches. There's probably something we can do, but would rather tackle it in a separate PR request so this one doesn't get too far reaching.